### PR TITLE
Batch renumber

### PR DESCRIPTION
**Pull request by ivandrofly**

### CODE REVIEW

**Comment by niksedk** [comment]
Nice - but I wonder if there could be any reason not to renumber?

**Comment by ivandrofly** [comment]
Can't find any... was requested by @miniop

**Comment by ivandrofly** [comment]
What you think about making plugin instead?

**Comment by niksedk** [comment]
OK, then perhaps the batch convert should just always renumber? (no GUI stuff needed)
@xylographe ?

**Comment by xylographe** [comment]
I can't think of _any_ reason (good or pedantic :wink: ) not to renumber.

I think, 'no GUI' is the best way to go. If someone were to complain later on, the GUI stuff could still be added.

File: src/Forms/BatchConvert.cs [hunk 1]
```diff
@@ -942,6 +949,10 @@ private static void DoThreadWork(object sender, DoWorkEventArgs e)
                     }
                 }
             }
+            if (p.Renumber)
+            {
+                p.Subtitle.Renumber(_startRenumFrom);
```
Comment (niksedk) [inline src/Forms/BatchConvert.cs]: Could you start with just this line?

Comment (ivandrofly) [inline src/Forms/BatchConvert.cs]: :+1:

**Comment by niksedk** [comment]
We start with the simple version: https://github.com/SubtitleEdit/subtitleedit/commit/8c90482b5f59aa9caf726cf152964941f6a606f4

**Pull request closed without merging**